beacon/goclient/goclient.go

```diff
@@ -75,13 +76,21 @@ func (gc *goClient) StartReceivingBlocks() {
 				gc.logger.Error("failed to fetch head block", zap.Error(err))
 				time.Sleep(time.Second * 2)
 				continue
+			} else if signedBeaconBlock == nil {
+				gc.logger.Debug("received empty block")
+				time.Sleep(time.Second * 1)
```

Comment:
@nivBlox why sleep? isn't it an error?

Reply:
clients in some cases returning empty block. no need special handling

Reply:
@nivBlox but why sleep?

Reply:
@alonmuroch to act similarly to error and success cases, both have time.sleep